workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/camunda/bpmn/CamundaBpmnBuilder.java

```diff
@@ -110,7 +109,12 @@ private void checkParentIsKnown(Map<String, String> parentActivities, String wor
   }
 
   private BpmnModelInstance workflowToBpmn(Workflow workflow) throws JsonProcessingException {
-    ProcessBuilder process = Bpmn.createExecutableProcess(createUniqueProcessId(workflow)).name(workflow.getId());
+    String processId = "";
+    if (workflow.getId() != null) {
```

Comment:
if the workflow id is not set, is it ok to have a process with an empty id? do the logs look ok in that situation?

Reply:
I put back the comment.
Technically at this level we can't have a workflow without id. workflowToBpmn() method is called by addWorkflow() method that is called by CamundaEnigne.deploy(). The latter sets workflow id if not provided.